thoth/user_api/api_v1.py

```diff
@@ -414,6 +414,30 @@ def list_python_package_indexes():
     return graph.python_package_index_listing()
 
 
+def list_hardware_environments(
+    is_external: bool = False,
+    page: int = 0,
+):
+    """List hardware environments in the graph database."""
+    graph = GraphDatabase()
+    graph.connect()
+    return {
+        "hardware_environments": graph.get_hardware_environments(is_external, page),
```

Comment:
I'm sending here as `page`, however I'm receiving it as `offset` at https://github.com/thoth-station/storages/pull/1134/files#diff-42370be78220d49029e40a3fbfc0f09fR2977. Is that OK?

Reply:
If it matches with other endpoints, its good :+1: (so we are consistent).